### ISSUE TITLE
Request: Enable exceptAfterSingleLine on lines-between-class-members

### ISSUE THREAD

**Issue by coreyfarrell** [xojs/eslint-config-xo#55]
I've started using [class properties](https://github.com/tc39/proposal-class-fields) in some of my browser code.  xo is complaining about code such as the following:
```js
class MyClass {
  x = 1;
  y = 1;
  z = 1;

  sum() {
    return x + y + z;
  }
}
```

IMO the above example code is exactly correct, no line feed between class field declarations but a linefeed before the `sum()` function.  Unfortunately it is not possible to exactly enforce this per https://github.com/eslint/eslint/issues/11153.  As a work-around I'm enabling `exceptAfterSingleLine` for my own projects.  This allows multiple fields to be declared without a line-feed between each, but unfortunately does not enforce having a line-feed between the fields and the first function.  This change would mostly have no effect on classes that do not use class fields as `xo` forbids non-empty functions from being single line.

Would you accept a patch enabling this option for the default xo config?
